docs/streams/developer-guide/processor-api.html

```diff
@@ -261,11 +262,13 @@ <h2>
                                 space.</li>
                             <li>RocksDB settings can be fine-tuned, see
                                 <a class="reference internal" href="config-streams.html#streams-developer-guide-rocksdb-config"><span class="std std-ref">RocksDB configuration</span></a>.</li>
-                            <li>Available <a class="reference external" href="/{{version}}/javadoc/org/apache/kafka/streams/state/Stores.html#persistentKeyValueStore-java.lang.String-">store variants</a>:
-                                time window key-value store, session window key-value store.</li>
-                            <li>Use <a class="reference external" href="/{{version}}/javadoc/org/apache/kafka/streams/state/Stores.html#persistentTimestampedKeyValueStore-java.lang.String-">persistentTimestampedKeyValueStore</a>
-                                when you need a  persistent key-(value/timestamp) store that supports put/get/delete and range queries.</li>
-                            <li>Use <a class="reference external" href="/{{version}}/javadoc/org/apache/kafka/streams/state/Stores.html#persistentTimestampedWindowStore-java.lang.String-java.time.Duration-java.time.Duration-boolean-">persistentTimestampedWindowStore</a>
+                            <li>Available <a class="reference external" href="/{{version}}/javadoc/org/apache/kafka/streams/state/Stores.html#persistentKeyValueStore(java.lang.String)">store variants</a>:
+                                versioned key-value store, time window key-value store, session window key-value store.</li>
```

Comment:
This part of the docs seems to be messy? We don't list timestampe-kv here but as own bullet point below. Below, session-store is missing as bullet point...

Reply:
Added timestamped KV, regular window stores (previously only timestamped window stores were represented), and session stores into this section, in order to be comprehensive. LMK if it's too much now.